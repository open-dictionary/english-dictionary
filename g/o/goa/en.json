{
	"word": "goa",
	"definitions": [
		{
			"definition": "a state of southwestern India",
			"pos": "noun"
		}
	]
}