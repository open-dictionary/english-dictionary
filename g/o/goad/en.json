{
	"word": "goad",
	"definitions": [
		{
			"definition": "a pointed instrument that is used to prod into a state of motion",
			"pos": "noun"
		},
		{
			"definition": "a verbalization that encourages you to attempt something",
			"pos": "noun"
		},
		{
			"definition": "give heart or courage to",
			"pos": "verb"
		},
		{
			"definition": "urge with or as if with a goad",
			"pos": "verb"
		},
		{
			"definition": "stab or urge on as if with a pointed stick",
			"pos": "verb"
		},
		{
			"definition": "annoy or provoke, as by constant criticism",
			"pos": "verb"
		}
	]
}