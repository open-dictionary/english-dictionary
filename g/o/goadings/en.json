{
	"word": "goadings",
	"definitions": [
		{
			"definition": "a verbalization that encourages you to attempt something",
			"pos": "noun"
		}
	]
}