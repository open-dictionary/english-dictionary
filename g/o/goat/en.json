{
	"word": "goat",
	"definitions": [
		{
			"definition": "any of numerous agile ruminants related to sheep but having a beard and straight horns",
			"pos": "noun"
		},
		{
			"definition": "a victim of ridicule or pranks",
			"pos": "noun"
		},
		{
			"definition": "(astrology) a person who is born while the sun is in Capricorn",
			"pos": "noun"
		},
		{
			"definition": "the tenth sign of the zodiac",
			"pos": "noun"
		}
	]
}