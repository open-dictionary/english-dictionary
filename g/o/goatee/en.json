{
	"word": "goatee",
	"definitions": [
		{
			"definition": "a small chin beard trimmed to a point",
			"pos": "noun"
		}
	]
}