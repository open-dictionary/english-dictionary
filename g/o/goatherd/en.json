{
	"word": "goatherd",
	"definitions": [
		{
			"definition": "a person who tends a flock of goats",
			"pos": "noun"
		}
	]
}