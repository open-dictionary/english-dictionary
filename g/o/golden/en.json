{
	"word": "golden",
	"definitions": [
		{
			"definition": "having the deep slightly brownish color of gold",
			"pos": "adjective"
		},
		{
			"definition": "marked by peace and prosperity",
			"pos": "adjective"
		},
		{
			"definition": "made from or covered with gold",
			"pos": "adjective"
		},
		{
			"definition": "supremely favored",
			"pos": "adjective"
		},
		{
			"definition": "suggestive of gold",
			"pos": "adjective"
		},
		{
			"definition": "presaging or likely to bring good luck or a good outcome",
			"pos": "adjective"
		}
	]
}